### ISSUE TITLE
Adjust location of game definition in "Collections"

### ISSUE THREAD

**Issue by k4y3ff** [cognitect/clojure-lab#14]
In [`collections.clj`](https://github.com/cognitect/clojure-lab/blob/fb701205553cd2a606013ddb538ba57d45145b3f/src/cljlab/collections.clj), some of the exercises refer to `game`, which is shown as being `def`'d earlier in the text.

However, `game` has not been defined unless the attendees _intentionally_ went back and eval'd that segment when they were reading it. As a result, some of the exercises make it seem like the function solutions should take three parameters instead of two--or something similar--because the error messages make it unclear that `game` should be globally defined.

tl;dr Either adjust the exercises to take `game` as a parameter, move the `game` definition into the same place as the exercises, or something else vaguely.
